login/service.go

```diff
@@ -339,7 +339,7 @@ func (keycloak *KeycloakOAuthProvider) checkFederatedIdentity(ctx context.Contex
 		log.Error(ctx, map[string]interface{}{
 			"err": err.Error(),
 		}, "Unable to crete http request")
-		return false, er.NewInternalError(errs.Wrap(err, "unable to crete http request"))
+		return false, er.NewInternalError(ctx, errs.Wrap(err, "unable to crete http request"))
```

Comment:
there is a typo here: `crete`. I thought I  already found it in your previous PR.